clients/java/client/src/it/java/org/camunda/bpm/client/client/ClientIT.java

```diff
@@ -233,6 +234,30 @@ public void shouldThrowExceptionDueToBaseUrlIsNull() {
     }
   }
 
+  @Test
+  public void shouldThrowExceptionDueToBaseUrlAndBaseUrlResolverIsNull() {
+    ExternalTaskClient client = null;
+
+    try {
+      // given
+      ExternalTaskClientBuilder externalTaskClientBuilder = ExternalTaskClient.create();
+
+      // then
+      thrown.expect(ExternalTaskClientException.class);
+
+      // when
+      client = externalTaskClientBuilder
+          .baseUrl(null)
+          .urlResolver(null)
```

Comment:
🙃  One more test case can be added when only null `urlResolver` is provided but it's also fine to stay like this.